app.py

```diff
@@ -127,53 +133,82 @@ def message(event, client):
-                # bot also records the originally scraped message and the timestamp of that scraped message.
+                        file_write(f"![Supplemental Image]({text})\n\n", wiki_title)
+
+                # bot writes response to question 3.
+                # bot also writes the originally scraped message and the timestamp of that scraped message.
                 # the timestamp is used to check whether that message has been scraped already. it is used in the function message_in_documentation().
                 elif last_question_asked == 3:
                     client.chat_postMessage(channel=channel_id, text=bot_messages.response_recorded, thread_ts=message_thread_ts)
                     client.chat_postMessage(channel=channel_id, text=bot_messages.wiki_page_made, thread_ts=message_ts)
                     scraped_message = thread_message_history[0]["text"]
                     if text == "None":
-                        file_write(f"## Original Documentation:\n\n{scraped_message}\n\n## Additional Info:\n\n###### Message Timestamp: {message_thread_ts}\n\n")
+                        file_write(f"## Original Documentation\n\n{scraped_message}\n\n## Additional Info\n\nMessage Timestamp: {message_thread_ts}\n\n", wiki_title)
                     else:
-                        file_write(f"## Original Documentation:\n\n{scraped_message}\n\n## Additional Info:\n\n{text}\n\n###### Message Timestamp: {message_thread_ts}\n\n")
+                        file_write(f"## Original Documentation\n\n{scraped_message}\n\n## Additional Info\n\n{text}\n\nMessage Timestamp: {message_thread_ts}\n\n", wiki_title)
+                    subprocess.call(['git', 'add', f"{wiki_title}.md"], cwd="../docs-slackbot.wiki")
+                    subprocess.call(['git', 'commit', '-am', f"A new wiki named {wiki_title} has been uploaded."], cwd="../docs-slackbot.wiki")
+                    subprocess.call(['git', 'push', '-f'], cwd="../docs-slackbot.wiki")
 
     # ========================= Code End ========================= #
 
-
-# a function that uses the timestamp of a message to check if that timestamp exists in the markdown file already.
+# indev, will be commented for now
+# the following listener will listen for the /scrape command and change the user privacy status based on the input.
+# @app.command("/scrape")
+# def user_privacy(ack, command, respond):
+#     ack()
+#     try:
+#         if command["text"].lower() == "on":
+#             respond("Other users can now scrape your messages.")
+#         elif command["text"].lower() == "off":
+#             respond("Other users will now be unable to scrape your messages.")
+#     # This is a KeyError. It occurs when the user did not input text alongside the command.
+#     # In other words, command["text"] returned no text.
+#     except KeyError:
+#         return respond("Please type on or off to change whether you want others to scrape your messages.")

```

Comment:
Try to avoid including commented out code in a PR

Reply:
fixed -- i kept my commented code somewhere else.